client/components/credit-card-form-fields/README.md

```diff
@@ -16,21 +16,22 @@ Some of these fields use [masking](https://en.wikipedia.org/wiki/Input_mask), i.
 ## Usage
 
 ```jsx
-import React from 'react';
+import React, { Component } from 'react';
 import CreditCardFormFields from 'components/credit-card-form-fields';
 
-React.createClass( {
-	render: function() {
+class YourComponent extends Component {
+	render() {
 		return (
 			<CreditCardFormFields
 				card={ this.props.card }
 				countriesList={ this.props.countriesList }
 				eventFormName="Credit Card Form"
 				isFieldInvalid={ this.isFieldInvalid }
-				onFieldChange={ this.onFieldChange } />
+				onFieldChange={ this.onFieldChange }
+			/>
 		);
 	}
-} );
+}
```

Comment:
If this was an actual component I'd probably ask for it be a pure functional one